src/12_scopes.py

```diff
@@ -19,12 +20,13 @@ def outer():
     y = 120
 
     def inner():
-        y = 999
+       y = print(999)
```

Comment:
Use the `nonlocal` keyword here Petar, this will allow the function to use the nonlocal variable y, like soo..
```
def inner():
        nonlocal y
        y = 999
```